dask_jobqueue/sge.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, queue=None, project=None, resource_spec=None, walltime=None,
         super(SGECluster, self).__init__(**kwargs)
 
         header_lines = ['#!/bin/bash']
-
+        # SGE header build
```

Comment:
General piece of advice: do your best to keep a PR focussed and avoid unrelated changes. It's fine to keep it like this for this PR.